packages/node_modules/@webex/webex-core/src/webex-core.js

```diff
@@ -24,6 +24,7 @@ import UserAgentInterceptor from './interceptors/user-agent';
 import WebexTrackingIdInterceptor from './interceptors/webex-tracking-id';
 import WebexUserAgentInterceptor from './interceptors/webex-user-agent';
 import RateLimitInterceptor from './interceptors/rate-limit';
+import EmbargoInterceptor from './interceptors/embargo';
```

Comment:
This properly loads the interceptor into the base package. [see other entries in this file]